chapters/1-rationale.md

```diff
@@ -88,4 +88,4 @@ Companies and organizations (collectively “Organizations”) are widely using
 
 **1.8.5** A new appendix “Using SPDX short identifiers in Source Files” has been added to document the best practices to refer to the licenses in the SPDX license list that have emerged from the development community. See [Appendix V: Using SPDX short identifiers in Source Files](./appendix-V-using-SPDX-short-identifiers-in-source-files.md) for more information.
 
-**1.8.6** Miscellaneous bug fixes as reported on the mailing list and reported as issues on the [spdx-spec GitHub repository](https://github.com/spdx/spdx-spec).
\ No newline at end of file
+**1.8.6** Miscellaneous bug fixes as reported on the mailing list and reported as issues on the [spdx-spec GitHub repository](https://github.com/spdx/spdx-spec).
```

Comment:
This should not have changed when changing only the year

Reply:
Agree.   It should not have been changed.

Reply:
Went in and did some manual edits on the pull.   Should be fine now.  If not, please flag as an issue.